examples/standalone_evaluation_llm_as_judge.py

```diff
@@ -0,0 +1,97 @@
+from unitxt import add_to_catalog, get_logger
+from unitxt.api import evaluate
+from unitxt.blocks import Task, TaskCard
+from unitxt.inference import (
+    HFPipelineBasedInferenceEngine,
+)
+from unitxt.llm_as_judge import LLMAsJudge
+from unitxt.loaders import LoadFromDictionary
+from unitxt.standard import StandardRecipe
+from unitxt.templates import InputOutputTemplate, TemplatesDict
+
+logger = get_logger()
+
+data = {
+    "test": [
+        {"question": "What is the capital of Texas?", "answer": ""},
+        {"question": "What is the color of the sky?", "answer": ""},
+    ]
+}
+
+rating_template = InputOutputTemplate(
+    instruction="Please act as an impartial judge and evaluate if the answer of the assistant is correct."
+    "Rate the response on a scale of 1 to 10, where 1 means totally wrong, and 10 means totally correct,"
+    ' by strictly following this format: "[[rating]]"'
+    ".\n\n",
+    input_format="[Question]\n{question}\n\n" "[Assistant's Answer]\n{answer}\n",
+    output_format="[[{rating}]]",
+    postprocessors=[
+        r"processors.extract_mt_bench_rating_judgment",
+    ],
+)
+
+add_to_catalog(
+    rating_template,
+    "templates.rating_template",
+    overwrite=True,
+)
+
+rating_metric = LLMAsJudge(
+    inference_model=HFPipelineBasedInferenceEngine(
+        model_name="google/flan-t5-large", max_new_tokens=256, use_fp16=True
+    ),
+    template="templates.rating_template",
+    task="rating.single_turn",
+    format="formats.models.mistral.instruction",
+    main_score="flan-t5-large_huggingface",
+    strip_system_prompt_and_format_from_inputs=False,
```

Comment:
Why is this required?

Reply:
Why should it be false.  Doesn't it work with 'true'?

Reply:
As we discussed, currently the code supports only rating.single_turn (and rating.single_turn_with_reference), The main score name has been changed. 
Regarding strip_system_prompt_and_format_from_inputs. Currently setting it to true caused the run to crash. I think we should solve it, but in a separate PR.

Reply:
Ok.